packages/smooth_app/lib/views/smooth_product_sneak_peek_view.dart

```diff
@@ -18,8 +18,7 @@ import 'package:smooth_app/cards/data_cards/smooth_energy_card.dart';
 import 'package:smooth_app/cards/data_cards/smooth_intake_recommendation_card.dart';
 import 'package:smooth_app/cards/data_cards/smooth_quantity_selector_card.dart';
 import 'package:smooth_app/data_models/sneak_peek_model.dart';
-import 'package:smooth_app/temp/product_extra.dart';
-import 'package:smooth_app/temp/available_attribute_groups.dart';
+import 'package:openfoodfacts/personalized_search/available_attribute_groups.dart';
 
 // TODO(stephanegigandet): remove if not useful anymore?
```

Comment:
There are several classes that can be deleted - maybe. All related to this one.
I added `// TODO(stephanegigandet)` comments on many of them. If I remember well, we both didn't exactly know what this particular class was all about, but maybe if you have a look at the "galaxy" of to-be-removed classes, that could make sense to you. And we can delete the classes. Or not, if relevant.